app/models.py

```diff
@@ -1295,6 +1296,7 @@ def serialize(self, with_users=False):
             'lot': self.lot.slug,
             'lotSlug': self.lot.slug,
             'lotName': self.lot.name,
+            'work_order_id': 0 if self.work_order is None else self.work_order.id,
```

Comment:
Convention seems to be to add these optional values in an if block after this initial dict and omit null values.